framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -205,6 +223,13 @@ private Instance createInstance(Pod pod) {
     protected Optional<URI> getWebSocketRouteUri(String serviceName) {
         return getRouteUri(Protocol.ws, serviceName);
     }
+    
+    protected RouteList getRoutes(){
+        return util.client()
+                   .routes()
+                   .withLabel("key", "value")
```

Comment:
key should be "service" and value should be getServiceName()

Reply:
sure.... fixed